Updated the Align and button pointer

Ratselaft opened the pull request.

beardedprince commented:
thanks. But this project already exist on the repo. Probably with the same team mate

adekunlemahmud committed:
Profile settings update

beardedprince committed:
Merge pull request #1 from adekunlemahmud/dev

Profile settings update

zonebyte committed:
Add files via upload

beardedprince committed:
stage 7 task

beardedprince committed:
Merge pull request #2 from zonebyte/dev

Add files via upload

Theresa-Akpan committed:
Navbar

Navbar for create estimate and invoice pages.

beardedprince committed:
Merge pull request #3 from Theresa-Akpan/dev

Navbar

Okikijoshua committed:
Files from my repo

mide358 committed:
create estimate page and guest create estimate page

OluSmartDev committed:
Merge pull request #1 from hngi/dev

Update Forked Repo

beardedprince committed:
Merge pull request #6 from mide358/dev

create estimate page and guest create estimate page

beardedprince committed:
Merge pull request #4 from Patrickodey360coder/dev

working on dashboard sidenav and invoice view sidebar

OluSmartDev committed:
Create settings-profile-navbars.html

Added Drop Down Menu to the 'Settings' contaning the following links:  Profile Settings, Email Settings and Subscriptions.

nkechi10 committed:
Pricing Update

Update of the Pricing page

beardedprince committed:
Merge pull request #8 from hngi/stage07-patch

Stage 7 patch

beardedprince committed:
Merge pull request #7 from oluscc/dev

Added settings-profile-navbars.html to the navbars folder

beardedprince committed:
Merge pull request #9 from nkechi10/dev

Pricing Update

Okikijoshua committed:
Pages

Okikijoshua committed:
pages

Vickyella23 committed:
worked on the new redirect page

Chelsea002 committed:
Made some fixes to the create estimate page.

I worked on the create estimate page (Evaluation Estimation) where i fixed the 'How much (in hours) do you charge per hour', by eliminating it as directed by the team lead. I also Fixed (coded using HTML5 and CSS3) the Navbar so it looks like the general navigation bar as displayed on the figma design. I finally fixed the NEXT button, at the footer to its appropriate size.

beardedprince committed:
Merge pull request #10 from Vickyella23/redirect

worked on the new redirect page

beardedprince committed:
Merge pull request #11 from Chelsea002/dev

Made some fixes to the create estimate page.

oladunniolajire committed:
Faq.html

Created the FAQ page

oladunniolajire committed:
Merge pull request #1 from oladunniolajire/oladunniolajire-patch-1

Faq.html

kadijatokeowo committed:
I worked on the navbar, the responsiveness and the heading message

epospiky committed:
modified create estimate page

beardedprince committed:
Merge pull request #13 from kadijatokeowo/seelect-client

I worked on the navbar, the responsiveness and the heading message

beardedprince committed:
Merge pull request #14 from epospiky/dev

modified create estimate page

beardedprince committed:
Merge pull request #12 from oladunniolajire/Branch_faq

Branch faq

Okikijoshua committed:
Review invoice page added, plus Favicon

Okikijoshua committed:
Favicon and review invoice page
https://github.com/hngi/newLancerApp into dev

beardedprince committed:
fixed hompage address as requested by xyluz

beardedprince committed:
Merge pull request #16 from hngi/homepage-fix

fixed hompage address as requested by xyluz

Okikijoshua committed:
review-invoice, favicon added in all pages

beardedprince committed:
Merge pull request #17 from Okikijoshua/dev

review-invoice, favicon added in all pages

Cheks-Cee committed:
made changes to faq.html

beardedprince committed:
Merge pull request #18 from Cheks-Cee/dev

made changes to faq.html

emmalegend committed:
changed nav and side bars, also made it responsive

Cheks-Cee committed:
Merge pull request #19 from emmalegend/emmalegend-task

changed nav and side bars, also made it responsive

Temidayo99 committed:
validation to the next buttons

Cheks-Cee committed:
Merge pull request #23 from Temidayo99/dev

validation to the next buttons

Cheks-Cee committed:
modified styling for company-detail.html

Cheks-Cee committed:
Merge pull request #26 from Cheks-Cee/dev

modified styling for company-detail.html

Akinnikemichael committed:
add and remove button

added add and remove button to add new contact name and contact email field

Akinnikemichael committed:
added add and remove button

for adding new field

Cheks-Cee committed:
Merge pull request #30 from Akinnikemichael/dev

add and remove button

Cheks-Cee commented:
No Image attached to this PR, can't merge
